staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2840,6 +2845,20 @@ const (
 	RestartPolicyNever     RestartPolicy = "Never"
 )
 
+// ContainerRestartPolicy is the restart policy for a single container.
+// Always is the only supported policy for now.
+type ContainerRestartPolicy string
+
+const (
+	// The container with ContainerRestartPolicyAlways will be restarted
```

Comment:
This is better on the field than here.

"""
Setting ContainerRestartPolicyAlways on an initContainer changes how its lifecycle is managed.  Kubernetes will still start this container in order with the other init containers, but instead of waiting for the container's completion, it will  wait for the container's startup completion (as indicated by ...).  If this container exits for any reason, it will be restarted, until all of the regular containers terminate, at which point this container will be terminated.  This is often referred to as a "sidecar" container.
"""

Reply:
moved it to the container's restartPolicy field.